client-app/shared/static-content/components/slider.vue

```diff
@@ -28,33 +30,66 @@
 </template>
 
 <script setup lang="ts">
+import { useBreakpoints } from "@vueuse/core/index";
 import { Navigation } from "swiper";
-import { Swiper, SwiperSlide } from "swiper/vue"; // eslint-disable-line import/no-unresolved
-import { computed, getCurrentInstance } from "vue";
-
-const props = defineProps<{
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  model: Record<string, any>;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  settings: Record<string, any>;
-}>();
-
-const componentId = `vc-slider_${getCurrentInstance()!.uid}`;
-
-const navigationOptions = {
-  prevEl: `#${componentId} .vc-slider__btn--prev`,
-  nextEl: `#${componentId} .vc-slider__btn--next`,
-  lockClass: "vc-slider__btn--lock",
-  hiddenClass: "vc-slider__btn--hidden",
-  disabledClass: "vc-slider__btn--disabled",
+import { Swiper, SwiperSlide } from "swiper/vue";
+import { computed } from "vue";
+import { BREAKPOINTS } from "@/core/constants";
+
+type BreakpointsType = keyof typeof BREAKPOINTS;
```

Comment:
Move to the same file where breakpoints specified

Reply:
done